app/build.gradle

```diff
@@ -8,12 +8,12 @@ plugins {
 
 android {
     namespace 'com.jshvarts.todoapp'
-    compileSdk 32
+    compileSdk 33
```

Comment:
needed for accompanist. I could get it working with older versions but took a path of least resistance